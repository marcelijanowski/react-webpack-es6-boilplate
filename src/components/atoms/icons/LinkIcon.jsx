import React, { PureComponent } from 'react';

class LinkIcon extends PureComponent {
  render() {
    return (<svg></svg>);
  }
}

export default LinkIcon;