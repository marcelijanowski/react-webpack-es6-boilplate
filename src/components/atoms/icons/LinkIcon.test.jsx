import React, { PureComponent } from 'react'
import renderer from 'react-test-renderer';

import LinkIcon from './LinkIcon';

describe('LinkIcon container', () => {
  it('renders a snapshot', () => {
    const tree = renderer.create(<LinkIcon/>).toJSON();
    expect(tree).toMatchSnapshot();
  });
})