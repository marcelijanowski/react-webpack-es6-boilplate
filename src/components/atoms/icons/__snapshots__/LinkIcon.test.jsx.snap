// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LinkIcon container renders a snapshot 1`] = `<svg />`;
